driver-core/src/test/java/com/datastax/driver/core/ShardAwareTest.java

```diff
@@ -165,4 +166,36 @@ public void verify_same_shard_in_tracing() throws InterruptedException {
     createData(session);
     this.queryData(session, true);
   }
+
+  @Test(groups = {"short", "scylla"})
+  @CCMConfig(
+      config = {
+        "ring_delay_ms:10000",
+        "native_transport_port:0",
+        "native_shard_aware_transport_port:19042"
+      })
+  public void verify_same_shard_in_tracing_using_only_shard_aware_port()
```

Comment:
This test is almost identical like the previous one. Why not have a helper method that runs the tests and just uses different port?